src/components/Alert/Alert.js

```diff
@@ -1,15 +1,9 @@
-import React, { Component, PropTypes } from 'react'
+import React, {Component, PropTypes} from 'react'
```

Comment:
we might want to think about importing from prop-types for v2. We dont need to do it here, so I will make an issue